templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,12 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 34**
+
+* [`POST /messages`]: Add option `moved_msgs_visible_to_all_subs` to
+  set whether to create user messages when moving messages to
+  private streams with protected history.
```

Comment:
We should explain this sort of feature in terms of what it means to users -- UserMessages aren't a concept for them.

"to control whether the messages should be added to the history of subscribers to the destination stream
when moving messages to a private stream with protected history".  We should have a more detailed explanation in `zulip.yaml`.

Reply:
Oh! Okay!